client/src/main.c

```diff
@@ -1099,9 +1099,14 @@ int eventFilter(void * userdata, SDL_Event * event)
           }
           break;
 
-        // allow a window close event to close the application even if ignoreQuit is set
+        // Allow a window close event to close the application even if ignoreQuit is set,
+        // but only when the cursor is not in the client area.
+        // On Wayland, keyboard shortcuts to kill windows trigger a close event, so they
+        // should be ignored if triggered while the mouse is in the VM.
```

Comment:
No need for the verbose comment, one can always look to git blame to see why this code was added. Also this should be the same for xorg anyway too :)

Reply:
I removed the whole comment since how the handler affects `ignoreQuit` should be clear from the code.